app/views/pages/home.html.erb

```diff
@@ -14,20 +14,18 @@
         Créez un carnet de bord digital pour votre nouveau petit héro !</p>
```

Comment:
It missing a paragraph tag : 
 <p> Créez un carnet de bord digital pour votre nouveau petit héro  ! </p>

Reply:
Hello, i fixed it ! Thank you :)